### ISSUE TITLE
it doesn't work on my mac i followed all instructions mac says "no such file"

### ISSUE THREAD

**Issue by bigray0x** [jcryptool/core#257]

**Comment by simlei** [comment]
Hi, 

the instructions you are referring to, are these, right?
> Open a "terminal" window (in Finder: Menu "Finder → Service → New terminal in folder"; or press "Control-Option-Command-T" while selecting the folder from its parent folder). Enter the following command there:
> `sudo xattr -r -d com.apple.quarantine ~/Downloads/jcryptool.app`
> (assuming you extracted JCrypTool directly in the folder "Downloads"; otherwise, adapt the command accordingly. The .app file ending should be kept.)

Could you please verify that your Downloads folder contains a file `jcryptool.app` after extracting the downloaded archive?

**Comment by bigray0x** [comment]
Yes that's it and the folder does contain the app after extraction 

Precisely ( application folder)

**Comment by simlei** [comment]
Could you try replacing `~/Downloads/jcryptool.app` in the command line with the full path to `jcryptool.app`? (and would you also share that path if you don't mind?)

**Comment by bigray0x** [comment]
Wish I could and I sorted it out
